docs/2023/DIGIT/Redesign and rewrite of MDMS/updates/2023-07-21.md

```diff
@@ -3,15 +3,23 @@ title: Week 3
 author: Limark Dcunha
 ---
 
-## Milestones
+## Milestoness
 
-- [ ] Give the description about Milestone 1
-- [ ] Give the description about Milestone 2
-- [ ] Give the description about Milestone 3
-- [ ] Give the description about Milestone 4
+- [x] Reconfigured the entire project up uptil now with DIGIT compatabile configurations
+- [x] Cretead a repository and raised PR's for the same
 
 ## Screenshots / Videos
 
+![Alt text](<../static/Week 3/Repo.JPG>)
+![Alt text](<../static/Week 3/PR_1.JPG>)
+![Alt text](<../static/Week 3/PR_2.JPG>)
+
 ## Contributions
 
+https://github.com/shailesh-egov/c4gt-mdms-redesign/pull/1
+
+https://github.com/shailesh-egov/c4gt-mdms-redesign/pull/2
+
```

Comment:
Try to keep the contributions and learnings in points and not just in paragraphs